app/controlplane/api/controlplane/v1/signing.proto

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+
+package controlplane.v1;
+
+option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/api/controlplane/v1;v1";
+
+service SigningService {
+  rpc SigningCert (SigningCertRequest) returns (SigningCertResponse);
+}
+
+message SigningCertRequest {
+  bytes certificate_signing_request = 1;
```

Comment:
btw, you can add here validations too, 

for example

```
  bytes certificate_signing_request = 1 [(buf.validate.field).bytes.min_len = 1];
```

https://buf.build/bufbuild/protovalidate/docs/main:buf.validate#buf.validate.BytesRules